gettsim/data/e_st_abzuege.yaml

```diff
@@ -1,56 +1,76 @@
-kifreib:
+kifreib_s_exm:
   name:
-    de: Kinderfreibetrag (sächliches Existenzminimum des Kindes sowie Betreuungs- und Erziehungs- oder Ausbildungsbedarf)
+    de: Kinderfreibetrag (sächliches Existenzminimum des Kindes).
     en:
   description:
     de: § 32(6) EStG
     en:
   values:
     1984-01-01:
-      value: 221
+      value: 110
       note:
     1986-01-01:
-      value: 1270
+      value: 635
       note:
     1990-01-01:
-      value: 1546
+      value: 773
       note: .
     1994-01-01:
-      value: 2098
+      value: 1049
       note:
     1996-01-01:
-      value: 3203
-      note: .
+      value: 1601
+      note: Art. 1. G. v. 20.10.1995 BGBl. I S. 1250
     1997-01-01:
-      value: 3534
+      value: 1767
       note: .
     2000-01-01:
-      value: 5080
-      note: .
+      value: 1767
+      note: Art. 1 G. v. 28.12.1999 BGBl. I S. 2552
     2002-01-01:
-      value: 5808
-      note:
+      value: 1824
+      note: Art. 1 G. v. 19.12.2000 BGBl. I S. 1790
     2009-01-01:
-      value: 6024
-      note:
+      value: 1932
+      note: Art. 1 G. v. 22.12.2008 BGBl. I S. 2955
     2010-01-01:
-      value: 7008
-      note:
+      value: 2184
+      note: Art. 1 G. v. 22.12.2009 BGBl. I S. 3950
     2015-01-01:
-      value: 7152
-      note: geändert durch BGBl. 2015 Nr. 30
+      value: 2256
+      note: Art. 1 G. v. 16.07.2015 BGBl. I S. 1202
     2016-01-01:
-      value: 7248
-      note: geändert durch BGBl. 2015 Nr. 30
+      value: 2304
+      note: Art. 2 G. v. 16.07.2015 BGBl. I S. 1202
     2017-01-01:
-      value: 7356
-      note: geändert durch Artikel 8 v. 20.12.2016, BGBl I S. 3000
+      value: 2358
+      note: Art. 8 G. v. 20.12.2016 BGBl. I. S. 3000
     2018-01-01:
-      value: 7428
-      note: Art. 9 BGBL I. S. 3000 v. 20.12.2016
+      value: 2394
+      note: Art. 9 G. v. 20.12.2016 BGBl. I. S. 3000
     2019-01-01:
-      value: 7620
+      value: 2490
+      note: Art. 1 G. v. 29.11.2018 BGBl. I S. 2210
+kifreib_bea:
+  name:
+    de: Kinderfreibetrag. Betreuungs-, Erziehungs-, Ausbildungsfreibetrag.
+    en:
+  description:
+    de: §32 (6) EStG, seit 2000.
+    en:
+  values:
+    1970-01-01:
+      value: 0
+      note: implizit
```

Comment:
is that really necessary? I have a hunch that it would rather improve things for the reader to make this clear in the source code (e.g. add a year switch). Would the source code look too complicated if we did not (mis)use this parameter?

Reply:
Agree. We need to have a standard way for dealing with parameters that are not around for certain years.

So rather have a check in the code for whether the parameter is there than to add zero.